dcpy/connectors/edm/recipes.py

```diff
@@ -246,6 +267,9 @@ def purge_recipe_cache():
 
 
 def log_metadata(config: library.Config):
+    if configuration.DEV_FLAG:
+        logger.info("DEV_FLAG found, not library run metadata")
```

Comment:
a bit confused by the meaning of this log message...

Reply:
Yeah that is fair